Mathlib/Algebra/Star/SelfAdjoint.lean

```diff
@@ -318,8 +318,21 @@ section Ring
 
 variable [Ring R] [StarRing R]
 
-instance : One (selfAdjoint R) :=
```

Comment:
I think it's probably best to define this first and then build the new instance out of it